Expected `cwd` to be of type `string` but received type `undefined`

<!-- Please answer the following. Issues that do not will be closed. -->

**Clearly describe the bug**
When trying to run stylelint, it breaks with the following error:
```shell
> react-redux-boilerplate@0.0.1 lint /Users/enrique-ramirez/Projects/react-redux-boilerplate
> stylelint src/**/*.css

TypeError: Expected `cwd` to be of type `string` but received type `undefined`
    at module.exports (/Users/enrique-ramirez/Projects/react-redux-boilerplate/node_modules/dir-glob/index.js:44:25)
    at globDirs (/Users/enrique-ramirez/Projects/react-redux-boilerplate/node_modules/globby/index.js:58:9)
    at getPattern (/Users/enrique-ramirez/Projects/react-redux-boilerplate/node_modules/globby/index.js:61:64)
    at Promise.all.globTasks.map.task (/Users/enrique-ramirez/Projects/react-redux-boilerplate/node_modules/globby/index.js:72:69)
    at Array.map (<anonymous>)
    at module.exports (/Users/enrique-ramirez/Projects/react-redux-boilerplate/node_modules/globby/index.js:72:41)
    at module.exports (/Users/enrique-ramirez/Projects/react-redux-boilerplate/node_modules/stylelint/lib/standalone.js:192:10)
    at Promise.resolve.then.then.options (/Users/enrique-ramirez/Projects/react-redux-boilerplate/node_modules/stylelint/lib/cli.js:511:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:182:7)
```
I tried to use a direct path (`src/file.css`) instead of a glob, but the results where the same. What's weird is I'm pretty sure I ran this like 10 minutes ago and it passed.

**Which rule, if any, is the bug related to?**
None

**What CSS is needed to reproduce the bug?**
None

**What stylelint configuration is needed to reproduce the bug?**
Not sure if it's related, but here's my configuration just in case:

```json
{
  "extends": "stylelint-config-standard",
  "plugins": [
    "stylelint-order",
    "stylelint-declaration-block-no-ignored-properties"
  ],
  "rules": {
    "at-rule-no-vendor-prefix": true,
    "block-no-empty": null,
    "color-no-invalid-hex": true,
    "comment-empty-line-before": [ "always", {
      "ignore": ["stylelint-commands", "after-comment"]
    } ],
    "declaration-colon-space-after": "always",
    "indentation": [2, {
      "except": ["value"]
    }],
    "max-empty-lines": 2,
    "media-feature-name-no-vendor-prefix": true,
    "no-missing-end-of-source-newline": 2,
    "order/properties-order": [["composes"], {"unspecified": "bottomAlphabetical"}],
    "plugin/declaration-block-no-ignored-properties": true,
    "property-no-unknown": [ true, {
      "ignoreProperties": ["composes"]
    } ],
    "property-no-vendor-prefix": true,
    "rule-empty-line-before": [ "always", {
      "except": ["first-nested"],
      "ignore": ["after-comment"]
    } ],
    "selector-no-vendor-prefix": true,
    "selector-pseudo-class-no-unknown": [ true, {
      "ignorePseudoClasses": ["global"]
    } ],
    "unit-whitelist": ["em", "rem", "%", "s", "px", "deg"],
    "value-no-vendor-prefix": true
  }
}
```

**Which version of stylelint are you using?**
9.9.0

**How are you running stylelint: CLI, PostCSS plugin, Node API?**
CLI with `stylelint src/**/*.css`

**Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?**
No

**What did you expect to happen?**
No errors and linter running

**What actually happened (e.g. what warnings or errors did you get)?**
See above.

Getting the same error here. I tried what you suggested @jeddy3 and deleted the node_modules. I was on Node 9.5 but I upgraded to 10.15.0 and still got the same problem.

**Clearly describe the bug**
I'm trying to run stylelint from my package.json scripts using (taking out the escaped quotes also breaks w/ same err): `"lint:css": "stylelint \"./src/*.css\""`

I get the following: 
```
> elevation-website@2.4.1 lint:css /Users/chrischuck/Desktop/elevation
> stylelint "./src/*.css"

TypeError: Expected `cwd` to be of type `string` but received type `undefined`
    at module.exports (/Users/chrischuck/Desktop/elevation/node_modules/dir-glob/index.js:44:25)
    at globDirs (/Users/chrischuck/Desktop/elevation/node_modules/globby/index.js:58:9)
    at getPattern (/Users/chrischuck/Desktop/elevation/node_modules/globby/index.js:61:64)
    at Promise.all.globTasks.map.task (/Users/chrischuck/Desktop/elevation/node_modules/globby/index.js:72:69)
    at Array.map (<anonymous>)
    at module.exports (/Users/chrischuck/Desktop/elevation/node_modules/globby/index.js:72:41)
    at module.exports (/Users/chrischuck/Desktop/elevation/node_modules/stylelint/lib/standalone.js:192:10)
    at Promise.resolve.then.then.options (/Users/chrischuck/Desktop/elevation/node_modules/stylelint/lib/cli.js:511:14)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! elevation-website@2.4.1 lint:css: `stylelint "./src/*.css"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the elevation-website@2.4.1 lint:css script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/chrischuck/.npm/_logs/2019-01-14T02_17_40_114Z-debug.log
```

**Which rule, if any, is the bug related to?**
None

**What CSS is needed to reproduce the bug?**
None

**What stylelint configuration is needed to reproduce the bug?**
```
{
  "extends": "stylelint-config-standard"
}
```

**Which version of stylelint are you using?**
9.9.0

**How are you running stylelint: CLI, PostCSS plugin, Node API?**
CLI (see above)

**Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?**
No

**What did you expect to happen?**
css linting magic

**What actually happened (e.g. what warnings or errors did you get)?**
no css linting magic :( (see above)

--- a/package.json
+++ b/package.json
@@ -48,7 +48,7 @@
     "file-entry-cache": "^2.0.0",
     "get-stdin": "^6.0.0",
     "global-modules": "^2.0.0",
-    "globby": "^8.0.0",
+    "globby": "^9.0.0",
     "globjoin": "^0.1.4",
     "html-tags": "^2.0.0",
     "ignore": "^5.0.4",

